ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidator.java

```diff
@@ -120,7 +120,7 @@ public ValidationResult<TransactionInvalidReason> validate(
       final ValidationResult<TransactionInvalidReason> blobTransactionResult =
           blobsValidator.validate(transaction);
       if (!blobTransactionResult.isValid()) {
-        LOG.info(
+        LOG.debug(
```

Comment:
would atDebug() save some processing here in the case where debug is not enabled